de.metas.vertical.pharma/src/main/java/de/metas/impexp/bpartner/PharmaImportPartnerInterceptor.java

```diff
@@ -51,19 +53,82 @@ private PharmaImportPartnerInterceptor()
 	@Override
 	public void onImport(IImportProcess<?> process, Object importModel, Object targetModel, int timing)
 	{
-		if (timing != IImportInterceptor.TIMING_AFTER_IMPORT || !(targetModel instanceof org.compiere.model.I_C_BPartner))
+		if (timing != IImportInterceptor.TIMING_AFTER_IMPORT )
 		{
 			return;
 		}
 
 		final I_I_BPartner ibpartner = InterfaceWrapperHelper.create(importModel, I_I_BPartner.class);
+
+		if (targetModel instanceof org.compiere.model.I_C_BPartner)
+		{
+			importBPartnerPharmaFields(ibpartner, targetModel);
+		}
+		else if (targetModel instanceof I_AD_User)
+		{
+			importContactPharmaFields(ibpartner, targetModel);
+		}
+	}
+
+	private void importBPartnerPharmaFields(@NonNull final I_I_BPartner ibpartner, @NonNull final Object targetModel)
+	{
 		final I_C_BPartner bpartner = InterfaceWrapperHelper.create(targetModel, I_C_BPartner.class);
 		bpartner.setIsPharmaciePermission(ibpartner.isPharmaciePermission());
 		if (!Check.isEmpty(ibpartner.getPharmaproductpermlaw52(),true))
 		{
 			bpartner.setPharmaproductpermlaw52(ibpartner.getPharmaproductpermlaw52());
 		}
+		//
+		if (!Check.isEmpty(ibpartner.getRegion(),true))
+		{
+			bpartner.setRegion(ibpartner.getRegion());
+		}
+		//
+		if (!Check.isEmpty(ibpartner.getSalesResponsible(),true))
+		{
+			bpartner.setPharmaproductpermlaw52(ibpartner.getPharmaproductpermlaw52());
```

Comment:
is this if with `SalesResponsible` and set with `Pharmaproductpermlaw52` correct?

Reply:
no! OMG, how did you see it?!

Reply:
luck, i guess :-)